qt/aqt/table.py

```diff
@@ -717,8 +717,10 @@ def _load_columns(self) -> None:
             ("note", tr.browsing_note()),
             ("noteCards", tr.editing_cards()),
             ("noteCrt", tr.browsing_created()),
+            ("noteDue", tr.statistics_due_date()),
```

Comment:
This is a bit tricky to follow at the moment, because we have the enum defined in rslib, the column labels defined in Python, and the mapping of names to the enum defined in rslib/backend. I would lean to moving this all into browser_table.rs, and exposing backend methods for the frontend to get the data it needs.